### DISCUSSION TITLE
Dockerfile stop after "mockoon-cli start"

### DISCUSSION THREAD

**Discussion by FedyaSavchuk** [mockoon/mockoon#530] [Help] [answered]
How I can use Dockerfile (without "mockoon-cli dockerize" command)

Like this:
```
FROM mockoon/cli:latest

COPY mockoon-stubs.json ./mockoon-stubs.json
EXPOSE 8082

ENTRYPOINT mockoon-cli start --data mockoon-stubs.json --name "Certificate"
```

**Comment by FedyaSavchuk** [comment, marked as answer]
I solved this problem. The main problem that I used "mockoon-cli" start instead of "mockoon-runner.sh"

My solution:

```
FROM mockoon/cli:latest
COPY mockoon-stubs.json ./data
COPY resources/ ./resources/

# Скрипт, который позволяет не спать mockoon
RUN echo "mockoon-cli start \"\$@\"; sleep infinity & wait \$!" > mockoon-runner.sh

EXPOSE 8082

ENTRYPOINT sh mockoon-runner.sh -d data -n 'Certificate'
```

**Reply by 255kb** [reply]
Yes, I guess if you want to build you own Dockerfile you must take example on the one in the repository. 
If you use the Docker image we published on the hub, you can pass the params at runtime: 
`docker run -d --mount type=bind,source=/home/your-data-file.json,target=/data,readonly -p 3000:3000 mockoon/cli:latest -d data -i 0 -p 3000`
You will find more info there: https://github.com/mockoon/cli#using-the-generic-docker-image
